homeworks/zsoltcsengeri/6_testing/test_bankaccount.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from bankaccount import BankAccount
+
+"""
+Write at least 2 fixtures (2 different BankAccount objects) that you use in your tests.
+Write at least one test that uses @pytest.mark.parametrize 
+to run the test with multiple inputs (e.g., test the deposit() 
+method with exactly 0 and negative number inputs)
+"""
+
+
+# Fixture for bankaccount_1 account
+@pytest.fixture
+def bankaccount_1():
+    # A BankAccount for Zsolt with $100 balance
+    return BankAccount("Zsolt", 100)
+
+
+# Fixture for bankaccount_2 account
+@pytest.fixture
+def bankaccount_2():
+    # A BankAccount for Maryna with $50 balance
+    return BankAccount("Maryna", 50)
+
+
+# Test for bankaccount_1 deposit
+def test_deposit_bankaccount_1(bankaccount_1):
+    bankaccount_1.deposit(600)
+    assert bankaccount_1.balance == 700
+
+
+# Test for bankaccount_2 deposit
+def test_deposit_bankaccount_2(bankaccount_2):
+    bankaccount_2.deposit(200)
+    assert bankaccount_2.balance == 250
+
+
+# Parametrized test for invalid deposit inputs for bankaccount_1
+@pytest.mark.parametrize(
+    "amount, expected_exception",
+    [
+        (-100, ValueError),  # Negative deposit
+        (0, ValueError),  # Zero deposit
+    ],
+)
+def test_deposit_invalid_input(bankaccount_1, amount, expected_exception):
```

Comment:
ez viszont már annyiból nagyobb gond, hogy ugyanaz a teszt neve ennek is, meg az alatta lévő tesztnek is ahol a bankaccount2-t teszteled. Elég csak az egyik objektumra megírni ezt a tesztet is.